exercises/trinary/trinary_test.rb

```diff
@@ -52,4 +52,33 @@ def test_invalid_trinary_with_digits_is_decimal_0
     skip
     assert_equal 0, Trinary.new('0a1b2c').to_decimal
   end
+
+  def test_invalid_trinary_with_multiline_string
+    skip
+    assert_equal 0, Trinary.new("Invalid\n201\nString").to_decimal
+  end
+
+  def test_number_out_of_range
+    skip
+    assert_equal 0, Trinary.new('4').to_decimal
+  end
+
+  # Problems in exercism evolve over time, as we find better ways to ask
+  # questions.
+  # The version number refers to the version of the problem you solved,
+  # not your solution.
+  #
+  # Define a constant named VERSION inside of the top level BookKeeping
+  # module.
+  #  In your file, it will look like this:
+  #
+  # module BookKeeping
+  #   VERSION = 1 # Where the version number matches the one in the test.
+  # end
+  #
+  # If you are curious, read more about constants on RubyDoc:
+  # http://ruby-doc.org/docs/ruby-doc-bundle/UsersGuide/rg/constants.htm
+  def test_bookkeeping
+    assert_equal 1, BookKeeping::VERSION
```

Comment:
Would version 2 be more appropriate? Or is there a convention that the unversioned version is always 0?